### PR TITLE
install vsa, adapt to new naming, launch image saver/stitcher, install new subpackage

### PR DESCRIPTION
**Pull request by fedorsc**

### CODE REVIEW

**Comment by lguyot** [comment]
Dear Fedor,

I ran the install script and it seems that everything went fine:

```
running install
running bdist_egg
running egg_info
writing requirements to vsa.egg-info/requires.txt
writing vsa.egg-info/PKG-INFO
writing top-level names to vsa.egg-info/top_level.txt
writing dependency_links to vsa.egg-info/dependency_links.txt
warning: manifest_maker: standard file 'setup.py' not found

reading manifest file 'vsa.egg-info/SOURCES.txt'
writing manifest file 'vsa.egg-info/SOURCES.txt'
installing library code to build/bdist.linux-x86_64/egg
running install_lib
running build_py
creating build/bdist.linux-x86_64/egg
creating build/bdist.linux-x86_64/egg/rosbridge_suite-develop
copying build/lib.linux-x86_64-2.7/rosbridge_suite-develop/__init__.py -> build/bdist.linux-x86_64/egg/rosbridge_suite-develop
byte-compiling build/bdist.linux-x86_64/egg/rosbridge_suite-develop/__init__.py to __init__.pyc
creating build/bdist.linux-x86_64/egg/EGG-INFO
copying vsa.egg-info/PKG-INFO -> build/bdist.linux-x86_64/egg/EGG-INFO
copying vsa.egg-info/SOURCES.txt -> build/bdist.linux-x86_64/egg/EGG-INFO
copying vsa.egg-info/dependency_links.txt -> build/bdist.linux-x86_64/egg/EGG-INFO
copying vsa.egg-info/requires.txt -> build/bdist.linux-x86_64/egg/EGG-INFO
copying vsa.egg-info/top_level.txt -> build/bdist.linux-x86_64/egg/EGG-INFO
zip_safe flag not set; analyzing archive contents...
creating 'dist/vsa-0.0.1-py2.7.egg' and adding 'build/bdist.linux-x86_64/egg' to it
removing 'build/bdist.linux-x86_64/egg' (and everything under it)
Processing vsa-0.0.1-py2.7.egg
Removing /home/lguyot/.local/lib/python2.7/site-packages/vsa-0.0.1-py2.7.egg
Copying vsa-0.0.1-py2.7.egg to /home/lguyot/.local/lib/python2.7/site-packages
vsa 0.0.1 is already the active version in easy-install.pth

Installed /home/lguyot/.local/lib/python2.7/site-packages/vsa-0.0.1-py2.7.egg
Processing dependencies for vsa==0.0.1
Searching for numpy==1.14.0
Best match: numpy 1.14.0
Adding numpy 1.14.0 to easy-install.pth file

Using /home/lguyot/.opt/platform_venv/lib/python2.7/site-packages
Finished processing dependencies for vsa==0.0.1
```



But when I run the CDP4 Experiment, I get:

 ```
 File "/home/lguyot/Documents/NRP/GazeboRosPackages/src/holographic/ros_holographic/nodes/visual_memory_module.py", line 12, in <module>
    from vsa.hrr import HRR
ImportError: No module named vsa.hrr
[visual_memory_module-5] process has died [pid 24318, exit code 1, cmd /home/lguyot/Documents/NRP/GazeboRosPackages/src/holographic/ros_holographic/nodes/visual_memory_module.py __name:=visual_memory_module __log:=/home/lguyot/.ros/log/825cd14e-1624-11e8-9c1e-f8b156cc0ab0/visual_memory_module-5.log].
log file: /home/lguyot/.ros/log/825cd14e-1624-11e8-9c1e-f8b156cc0ab0/visual_memory_module-5*.log
Couldn't import dot_parser, loading of dot files will not be possible.
```

**Review by lguyot — COMMENTED** [review]
I know that is unrelated to this particular change, but may be it will be useful for the next commit:
- I think we should use $NRP_VIRTUAL_ENV and not $VIRTUAL_ENV because the last one can be modified by some NRP build scripts
- "git pull" requires further arguments to succeed (maybe "origin master")

File: image_to_saccade.py [hunk 1]
```diff
@@ -0,0 +1,51 @@
+import rospy
+from sensor_msgs.msg import Image
+from geometry_msgs.msg import Point
+from cv_bridge import CvBridge
+
+from attention import Saliency, Saccade
+
+tensorflow_path = rospy.get_param("~tensorflow_path", "/opt/tensorflow_venv/lib/python2.7/site-packages")
```
Comment (jackokaiser) [inline image_to_saccade.py:8]: so tensorflow does not need to be in the pythonpath, it will be added here?

File: image_to_saccade.py [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+    if last_time.value is None:                                                                                                                                                                                           
+        last_time.value = t
+    current_time = t
+    dt = current_time - last_time.value
+    last_time.value = current_time
+
+    image = bridge.value.imgmsg_to_cv2(image.value, "bgr8")
+
+    saliency_map = saliency.value.compute_saliency_map(image)
```
Comment (jackokaiser) [inline image_to_saccade.py:39]: cool well done

File: image_to_saccade.py [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+    (target, is_actual_target, V, M) = saccade.value.compute_saccade_target(saliency_map, dt)
```
Comment (jackokaiser) [inline image_to_saccade.py:40]: V and M: I guess membrane potential, and..?
You should respect the naming convention and call it membrane_potential or potential or something, but capital letters are for classes

**Pull request merged**